### ISSUE TITLE
Filling slots with multi-word entities or free text (title, description, opinion ...)

### ISSUE THREAD

**Issue by d4vidbiertmpl** [RasaHQ/rasa_nlu#1954]
**Rasa Core version**: 8.3

**Python version**: 3.6

**Operating system** (windows, osx, ...): Ubuntu 17.10

**Issue**: Question

Hi there, 
Unfortunately, I do not have an issue to report, but a question which might be interesting for other's as well. 
 
I am currently working on a chatbot which should interview people on ideas they have. Therefore, they should describe the idea by giving it a title, description, keywords, ... etcetera. These are the slots I want to fill, but most of the titles and descriptions will consist of more than one word. In terms of the description, even a few sentences. Is this even considered an entity then? Moreover, the values are highly ambiguous, as those are not finite as in the restaurant example (e.g cuisine=[chinese, Italian... ]). A title for an idea could be anything and keywords may clash with words given in the description and vice versa.

So basically, I encountered the problem that regular entity extraction doesn't sufficiently work to fill those slots, as both intents and entities are inherently ambiguous. However, are there any known techniques for incorporating multi-word entities like a title or even a free text? From my understanding of machine-learning tools, I'm a bit puzzled how a model for detecting this would even look like... I actually thought about that a lot, even discussed with friends of mine who focus on NLP in their Master's.

If anyone can think of an approach for dealing with this or had a similar problem before, I would be grateful for any input. :)

Update: My current approach would be using Interactive Dialogues with text-fields and text areas and fill the slots with those.
